README.md

```diff
@@ -37,3 +37,134 @@ docker run -it --rm \
   -v $(pwd)/boardswarm/share/server.conf:/etc/boardswarm/server.conf \
   ghcr.io/boardswarm/boardswarm:main
 ```
+
+## Manual Installation
+
+These instructions are for configuring a simple unencrypted setup with static
+JWKS based authentication.
+
+### Building
+
+Boardswarm needs rustc version 1.70 or newer, this rules out a number of stable
+distributions for now (the kind of thing you may want to run your
```

Comment:
It only rules out the rustc available in the distro itself though (which sure fair), but lots of folks end up installing rust with rustup in which case it works well (or do the build in a container as you don't want dev dependencies on your server)

This paragraph is a bit convoluted because it assumes building with a rustc provided by eg.. debian though that's not entirely clear :)..

Fwiw to avoid people have to install rust etc i've been playing with creating pre-build binaries (next to the containers) (see the dist branch), unfortunately github makes that a tad awkward as you can't easily publish artifacts on a brnach without making a release

Reply:
As it turns out, the path I'd suggested it seems can also break if the libc on the building machine and the one on the target machine are sufficiently different:
```
boardswarm[54132]: /usr/local/bin/boardswarm: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38' not found (required by /usr/local/bin/boardswarm)
```

Reply:
Significantly reworked building instructions.